package.json

```diff
@@ -24,56 +24,56 @@
   },
   "dependencies": {
     "async": "0.2.7",
-    "aws-sdk": "^2.2.4",
+    "aws-sdk": "^2.23.0",
     "bestlang": "0.0.1",
-    "body-parser": "^1.13.1",
-    "chokidar": "^1.4.2",
+    "body-parser": "^1.17.1",
+    "chokidar": "^1.6.1",
     "colors": "^1.1.2",
-    "compression": "^1.5.0",
-    "cookie-parser": "^1.3.5",
+    "compression": "^1.6.2",
+    "cookie-parser": "^1.4.3",
     "cookie-session": "^1.1.0",
     "cookies-js": "~1.2.1",
-    "cors": "^2.7.1",
-    "cryptr": "^1.0.0",
-    "csurf": "^1.8.3",
-    "express": "^4.13.0",
+    "cors": "^2.8.1",
+    "cryptr": "^2.0.0",
```

Comment:
are we sure this is safe? minor version bumps (assuming correct `semver` use) are supposedly safe but major version bumps are API-compatibility-breaking changes.

Let's keep this at `^1.0.0` for now and file a separate issue for cryptr specifically. Going up major version numbers is worth "doing on its own"